subprojects/docs/src/docs/userguide/running-builds/gradle_daemon.adoc

```diff
@@ -219,7 +219,7 @@ Gradle uses operating system features for watching the file system.
 It supports the feature on these operating systems:
 
 - Windows 10,
-- Linux (Ubuntu 16.04 or later),
+- Linux (Ubuntu 16.04 or later, CentOS 8 or later, Red Hat Enterprise Linux 8 or later, Amazon Linux 2),
```

Comment:
Are we sure about Amazon Linux 2? This says it's based on but it's its own distro now: https://serverfault.com/questions/798427#comment1011304_798428

Amazon doesn't claim compatibility in their documentation either. I'd feel better if we left it out.

Reply:
We are running the native platform tests on Amazon Linux 2 already, and we build the ncurses6 variant there.

Reply:
In that case: 👍.